src/responsibleai/rai_analyse/_score_card/_rai_insight_data.py

```diff
@@ -15,6 +15,10 @@
 from datetime import datetime
 
 
+class UserConfigError(Exception):
+    pass
```

Comment:
Why do we need pass here? it is class definition

Reply:
Can you use UserConfigValidationException from raiutils.excpetions?

Reply:
It's actually a copy-paste of the same definition in the parent folder. I just can't reference it here because it's not a proper python package so `..` doesn't mean anything. [... which, of course, raises the question why it isn't. I don't know! I feel like it should be... certainly not in scope for this PR, though.]

`raiutils` isn't a depedency, but `responsibleai` and `raiwidgets` are, of course. So I'll switch over to UserConfigValidationException.